intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,78 @@
+/* global $ */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
+        var chartElement = $("#members-chart");
+        chartElement.show();
+        var canvas = chartElement[0];
+        var parsedData = [];
+        $.each(raw_data, function(k, v) {
+            parsedData.push([Date.parse(k, "YYYY-MM-DD"), v]);
+        });
+        parsedData = parsedData.sort(function(a, b) {
+            if (a[0] < b[0]) {
+                return -1;
+            }
+            if (a[0] > b[0]) {
+                return 1;
+            }
+            return 0;
+        });
+        var data = {
+            labels: $.map(parsedData, function(v) { return v[0]; }),
+            datasets: [
+                {
+                    label: "A Block",
+                    backgroundColor: "rgba(151,187,205,0.5)",
+                    data: $.map(parsedData, function(v) { return "A" in v[1] ? v[1].A : 0; })
+                },
+                {
+                    label: "B Block",
+                    backgroundColor: "rgba(205,187,151,0.5)",
+                    data: $.map(parsedData, function(v) { return "B" in v[1] ? v[1].B : 0; })
+                }
+            ]
+        };
+        var chart = new Chart(canvas.getContext("2d"), {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['chart' is defined but never used.](https://www.codacy.com/app/tjcsl/ion/file/2623655726/issues/source?bid=3176160&fileBranchId=3299251#l35)